### ISSUE TITLE
can't get it installed with the script and steps shown

### ISSUE THREAD

**Issue by rvdsv** [whiskerz007/proxmox_hassos_install#105]
tried the script several times but without any luck.

Even with a fresh install I can't get it to work maybe it is plain simple..

When I install the script and run the steps, after startup I get the following message.

[    6.964468] udevd[205]: bind failed: Address in use
[    6.965213] udevd[205]: error binding udev control socket

searched the internet but can't find it any more.

all help is welcome :)

**Comment by jrummy333** [comment]
I tried as well but it's no longer working.

**Comment by autumnwalker** [comment]
Having the same problem as @rvdsv. I did an install with this on a different box about four weeks ago and it worked fine.

**Comment by jrummy333** [comment]
I tried last night and fail but will try again in a bit
WHOOPS! Replied from email with my signature lol 
________________________________

**Comment by rvdsv** [comment]
have it running now on a different way.
used the install methode on the home-assistant.io website with qcow2. took me a couple of tries but have it running now in proxmox.

**Comment by jrummy333** [comment]
Could you elaborate a bit? I will need to follow your footsteps here as my system is down atm. 



have it running now on a different way.
used the install methode on the home-assistant.io website with qcow2. took me a couple of tries but have it running now in proxmox.

**Comment by rvdsv** [comment]
No problem.

Used the instructions on home assistant website https://www.home-assistant.io/hassio/installation/ ,download the qcow2 file, scroll down to the Proxmox installation guide and follow that.

But change the machine type to q35 as per instructions of this youtube video https://www.juanmtech.com/install-proxmox-and-virtualize-home-assistant/  use filezilla as shown.

I had to wait longer than the 20 mins of installation time. about an hour.

Hope this helps

PS I can see you signature with full email address... maybe better respond on github self.

**Comment by jrummy333** [comment]
Yep, I noticed lol  thanks for feedback though.  Need to get this back up and running soon :P

**Comment by autumnwalker** [comment]
I did the same as @rvdsv - the only thing that got me (and this was completely my fault) is I was trying to "add" the qcow2 drive to the virtual machine instead of "edit"ing the qcow2 drive to add it to the VM. I also had to go into "options" and set the "boot order" so that the qcow2 drive was selected and first in the list.

**Comment by ironicbadger** [comment]
> But change the machine type to q35 as per instructions of this youtube video https://www.juanmtech.com/install-proxmox-and-virtualize-home-assistant/ use filezilla as shown.

Why q35?

**Comment by jrummy333** [comment]
Apparently, something in the latest revision caused this to happen so you can still run the script using the previous repo and everything runs as expected.   At least for me, it worked fine.  Hope this helps!

bash -c "$(wget -qLO - https://raw.githubusercontent.com/whiskerz007/proxmox_hassos_install/4410a191334b341eb5621c3a4aa9b91b39fd63a6/install.sh)"

This will leave you with version 4.xx of HASSIO which you can just update from there.
